h2o-app/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
   if (project.hasProperty("doIncludeMojoPipeline") && project.doIncludeMojoPipeline == "true") {
     compile project(":h2o-ext-mojo-pipeline")
   }
+  compile project(":h2o-persist-gcs")
```

Comment:
Intentional. Let's keep it there for easier development, may I ?

Reply:
what about the other persists, should they be here as well? is there a reason they are not and are added only to assembly?

Reply:
I have no idea about the reason. @michalkurka please ?

Reply:
For now, I removed it from the dependencies.